### DISCUSSION TITLE
Test fixtures for sets?

### DISCUSSION THREAD

**Discussion by simivar** [rectorphp/rector#4843] [Q&A] [answered]
As you know, we have a few general Rectors that are used in sets with a call to `configure` to rename class constants, methods etc. Would it make sense to create some way of testing this to ensure that the changes for a set are done right?

**Comment by TomasVotruba** [comment, marked as answer]
The rules are already tested one by one.
Testing sets would only duplicate this logic. 

Also performance heavy, as each sets new container and that's ~3 seconds to boot.

We already have CI workflow that loads sets. It validates service syntax. Check GitHub Actions we have here.


Maybe rephrase.... What issue is happening that this would cover?

**Comment by TomasVotruba** [comment]
Closing as anwered
